### PR TITLE
Create LICENSE.md

### PR DESCRIPTION
**Pull request by Costallat**
Add a License to the project

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (gpedro) [inline LICENSE.md]: missing some replacements, uhm?

Comment (Costallat) [reply LICENSE.md]: Ops, sorry, I used the Github and did "next next next finish".

Shame on me!

Comment (gpedro) [reply LICENSE.md]: hahahah

File: LICENSE.md [hunk 2]
```diff
@@ -631,8 +631,8 @@ to attach them to the start of each source file to most effectively
 state the exclusion of warranty; and each file should have at least
 the "copyright" line and a pointer to where the full notice is found.
 
-    <one line to give the program's name and a brief idea of what it does.>
-    Copyright (C) <year>  <name of author>
+    OTServBR-Global - A free and open-source MMORPG server emulator written in C++
+    Copyright (C) 2019  Opentibiabr
```
Comment (gpedro) [inline LICENSE.md]: `Opentibiabr` > `OpenTibiaBR`

about software license, we need to add a third-party license too?

Comment (Costallat) [reply LICENSE.md]: I'm not expert in this License world, but I did a search here and found that they say as long as we keep an equal or compatible license with the third-party software we don't need to include it, TFS uses GPLv2 and OTX doesn't have a license, but as it is TFS based, it should have also the GPLv2 license. 

I also checked the TFS license and saw that someone did the same mistake that I did here, they only added the license without changing the program name, author and year.

Comment (Costallat) [reply LICENSE.md]: OTX -> https://github.com/malucooo/otxserver-new

the other is under GPLv2

https://github.com/mattyx14/otxserver

**Pull request merged**